netlify/functions/transDim.js

```diff
@@ -0,0 +1,167 @@
+      console.log(event);
+      return {
+         statusCode: 200,
+         headers: {
+            "Access-Control-Allow-Origin": transDimConfig.allow_domain,
+            "Access-Control-Allow-Methods": "POST",
+            "Access-Control-Allow-Headers":
+               "Origin, X-Requested-With, Content-Type, Accept"
+         }
+      };
+   }
+   // Handle responses to POST requests
+   else {
+      console.log(event);
+
+      // Decode the JSON from the request body string
+      const bodyParsed = JSON.parse(event.body);
+      const { formData } = bodyParsed;
+
+      const enquirerName = formData.name;
+      const enquirerEmail = formData.email;
+      const enquirerMessage = formData.message;
+
+      const fields = [
+         { label: "Name", value: enquirerName },
+         { label: "Email", value: enquirerEmail },
+         { label: "Phone number", value: formData.phone },
+         { label: "Job title", value: formData.job },
+         { label: "Organisation", value: formData.organisation },
+         { label: "Postcode", value: formData.postcode },
+         {
+            label: "I'd like a ring back",
+            value: formData.ringBack == true ? "Yes" : "No"
+         },
+         {
+            label: "I'd like more information",
+            value: formData.moreInfo == true ? "Yes" : "No"
+         }
+      ];
+
+      // Deconstruct the recipients and styling from the config for ease of use
+      const emailRecipients = transDimConfig.email_recipients;
+      const emailStyling = transDimConfig.email_styling;
+
+      // Construct the email from the request body
+      const email = {
+         priority: "normal",
+         from: {
+            name: enquirerName,
+            address: process.env.EMAIL_FROM // the EnquiryWitch email to send from
+         },
+         replyTo: {
+            name: enquirerName, // 'reply to' name details
+            address: enquirerEmail // 'reply to' email address
+         },
+         to: emailRecipients.info,
+         subject: `Request to join The Trans Dimension from ${enquirerName}`,
+         text:
+            `${emailStyling.header_text || "Web Enquiry"}\n\n-----------\n\n` +
+            fields
+               .map(
+                  (field) =>
+                     `${field.label}: ${
+                        field.value == "" ? "Not answered" : field.value
+                     }`
+               )
+               .join("\n") +
+            "\nMessage\n\n" +
+            enquirerEmail +
+            "\n\n-----------\n\nReply to this email to reply to the sender.\n\n-----------", // plain text email
+         html:
+            '<html>\n<body>\n<div id="email-container" style="background-color: #F6F6F6; height: 100%; width: 100%; display: block; padding: 25px 0;">\n<div id="message-container" style="display: block; background-color: white; width: 100%; max-width: 700px; margin: auto;">\n<div id="message-header" class="banner" style="background-color: #' +
```

Comment:
I wonder where this styling comes from - and if it is necessary? It doesn't seem particularly related to TD.

Reply:
see earlier comment